indexer/README.md

```diff
@@ -1,226 +1,176 @@
+
+### 4. Modifying IAM Access Policies
+
+1. In the AWS Console, click Services in the top toolbar, and click IAM in the Security, Identity & Compliance subsection.
+1. On the side menu bar, click **Roles**, then search for your Lambda function, `<your-indexer-lambda-application-name>`. Once you find it, click on it.
+1. Find a table in the Permissions tab under the blue Attach Policy button. You should see for Lambda name in the table. Click on the caret on the right side of the Lambda name. Then, click on "Edit Policy".
+1. Add the policy found in `policies/elasticsearch-policy.json` file in the cgp-dss-azul-indexer repository (Make sure to change the `Resource` value to the ARN of your elasticsearch box).
+
+### 5. Deploying Indexer Lambda
+
+1. In the directory of your chalice function, run
+   ```
+   chalice deploy --no-autogen-policy
+   ```
+   **Note:** If you are deploying an AWS API Gateway stage other than the default (dev)
+add the `--stage <stage name>` option. Since we have created a policy in AWS we do not want chalice to automatically create a policy for us.
+
+### 6. Registering a Subscription for Notification with the DSS
+
+1. Make sure your computer has a working web browser.
+1. If you are currently in a virtual environment, run the command `deactivate` to get out of the environment.
+1. Create a new Python 3.6 virtual environment and activate it.
+1. Install HCA using the command
+   ```
+   pip install --upgrade hca
+   ```
+1. Setting up config file using the instructions located [here](https://github.com/HumanCellAtlas/dcp-cli#development).
+1. Check if your HCA DSS deployment has been configured correctly by running
+   ```
+   hca dss post-search --replica aws --es-query '{}'
+   ```
+You should receive a response json containing one or more urls. Check if the returned URLs match the intended DSS Server.
+1. Login to HCA using the command
+   ```
+   hca dss login
+   ```
+1. You should have received the response `Please visit this URL to authorize this application...` and the corresponding URL.
+Click on link. Your web browser should appear requesting you to login. Login with your UCSC Gmail Account.
+1. Registering the subscription
+   ```
+   hca dss put-subscription --replica <aws or gcp> --callback-url <azul-indexer-url> --es-query '{"query": {"match_all": {}}}'
+   ```
+
+    - `<aws or gcp>`: your preferred storage bucket service
+    - `<azul-indexer-url>`: your azul indexer's url. If you need to know what is your url, following the directions 
+    in the below section *Finding the URL of your Azul Indexer Lambda*.
+1. Copy down the uuid for the subscription.
+
+## Reindexing
+
+Run the command below.
+```
+python test/find-golden-tickets.py --dss-url https://<your DSS URL>c--indexer-url https://<your elasticsearch endpoint> --es-query {}`
+```
+where you replace the `<>` value with your own values.
+- `<your DSS URL>` is the endpoint of the HCA DSS.
```

Comment:
Please add as an example a DSS URL that is properly configured/working for us.
There is often confusion about whether the URL schema (e.g. `https://`) and the API version (e.g. `/v1`) should be included or not.